docs/technology_models/grid.md

```diff
@@ -0,0 +1,70 @@
+# Grid Performance and Cost Models
+
+This page documents the unified `grid_performance` and `grid_cost` models, which together represent a flexible, configurable grid interconnection point within an H2I simulation.
+These components support both power flows and cost accounting for buying and selling electricity through a constrained interconnection.
+This is a single model that can be configured to either sell electricity to the grid, buy electricity from the grid, or both.
+
+See `example/23_solar_battery_grid` to see how to set up both buying and selling grid components.
+
+## Grid Performance
+`grid_performance` represents a grid interconnection point that can buy or sell electricity subject to a maximum throughput rating (interconnection_size).
+
+It supports:
+- Buying electricity from the grid to meet downstream demand.
+- Selling electricity to the grid.
+- Enforcing maximum allowed interconnection power.
+- Computing unmet demand and unsold electricity due to constraints.
+
+```{note}
+Multiple grid instances may be used within the same plant to represent different interconnection nodes. In the `tech_config` the component **must** be called `grid_buy` for the logic to work appropriately in financial calculations.
```

Comment:
Does this mean that every instance of `grid` mush be called `grid_buy`? or is that just for buying? Can the name just **include** `grid_buy` (e.g. `grid_buy_1`), or does it have to be exactly `grid_buy` with no additional elements to the name?

Reply:
Yes, the name must be `grid_buy` exactly. Fixing this in a general way is captured in #374